resources/core/charts/monitoring/charts/prometheus/values.yaml

```diff
@@ -128,6 +128,13 @@ resources:
   limits:
     memory: 2Gi
 
+isLocalEnv:
```

Comment:
for that isLocalEnv local variable, please use installer local and cluster overrides